org.eclipse.paho.android.service/src/main/java/org/eclipse/paho/android/service/AlarmPingSender.java

```diff
@@ -84,9 +87,17 @@ public void stop() {
 
 		Log.d(TAG, "Unregister alarmreceiver to MqttService"+comms.getClient().getClientId());
 		if(hasStarted){
+<<<<<<< HEAD
```

Comment:
Looks like you've got some merge data still in the file, could you remove this please?